CONTRIBUTING.md

```diff
@@ -16,6 +16,10 @@ limitations under the License.
 
 ## Contributing to AMP HTML
 
+The AMP HTML project strongly encourages technical contributions!
```

Comment:
Makes sense to me; they'll refer to each other now so people who start at one will know about the other.